spec/requests/conversion_hosts_spec.rb

```diff
@@ -53,9 +53,11 @@
   context "create" do
     let(:zone) { FactoryBot.create(:zone) }
     let(:ems_openstack) { FactoryBot.create(:ems_openstack, :zone => zone) }
-    let(:ems_redhat) { FactoryBot.create(:ems_redhat, :zone => zone) }
+    let(:ems_redhat) { FactoryBot.create(:ems_redhat, :zone => zone, :api_version => '4.2.4') }
```

Comment:
There's an `:ems_redhat_v4` factory, is `4.2.4` important?

Reply:
@bdunne We have a minimum api version for redhat conversion hosts of 4.2.4, and the factory is 4.0.

Reply:
I imagine the factory could probably be updated, but didn't want to touch it for now since it could affect tests outside of v2v.

Reply:
> @bdunne We have a minimum api version for redhat conversion hosts of 4.2.4, and the factory is 4.0.

Okay.

> I imagine the factory could probably be updated, but didn't want to touch it for now since it could affect tests outside of v2v.

No, I think that makes it even less obvious that this api version is important here.